packages/react-components/react-components/src/Concepts/Icons/IconTable.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from '@storybook/addon-docs';
+import { Spinner } from '@fluentui/react-spinner';
+
+<Meta title="Concepts/Developer/Icons/IconTable" />
+
+### Available Icons
+
+<React.Suspense fallback={<Spinner label="Loading..." />}>
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/183605527-17a4b35d-c351-4537-a87b-90e1134afdff.png)

Can we add some description there to clarify what users are expected to do?

````
This page contains a list of available icons under `@fluentui/react-icons` package. To use icons install the package and select an icon to use.

For example:

```tsx
import { DismissSquareRegular } from `@fluentui/react-icons`

function App() {
  return <DismissSquareRegular />
}
```
````